rs_workflows/staging.py

```diff
@@ -92,7 +93,12 @@ def update_stac_catalog(  # pylint: disable=too-many-locals
     now = datetime.now()
 
     # The file path from the temp s3 bucket is given in the assets
-    assets = {"file": Asset(href=f"{obs.rstrip('/')}/{stac_file_info['id']}")}
+    if kwargs.get("update_assets", None):
+        assets = {}
+        for asset in stac_file_info["assets"]:
+            assets.update({asset: Asset(href=f"{obs.rstrip('/')}/{stac_file_info['assets'][asset]['href']}")})
+    else:
+        assets = {"file": Asset(href=f"{obs.rstrip('/')}/{stac_file_info['id']}")}
```

Comment:
@Padeanu in case of several assets, your fix is OK, we will have e.g.
```json
  "assets": {
    "zarr": {
        // ...
    },
    "cog": {
        // ...
    }
  },
``` 

But in case of a single asset, we will have:
```json
  "assets": {
    "file": {
        // ...
    },
``` 

I've spoken with @vprivat-ads and it's not OK, we want the file type instead of "file" e.g:
```json
  "assets": {
    "zarr": {
        // ...
    },
```